views/js/qtiCreator/editor/response/choiceSelector.js

```diff
@@ -66,27 +66,27 @@ define([
         var choices = this.config.interaction.getChoices();
         var response = this.config.interaction.getResponseDeclaration();
         var config = _.defaults(this.config || {}, _defaults);
-        
+
         config.multiple = response.isCardinality(['multiple', 'ordered']);
         config.options = [];
 
-        _.each(choices, function(choice) {
+        _.some(choices, function(choice) {
             var id = choice.id();
             var choiceText = '';
             var option = {
                 value: id,
                 label: id,
                 selected: selected.indexOf(id) > -1
             };
-            
+
             if(choice.is('containerChoice')){
                 choiceText = choice.body();
             }else if(choice.is('textVariableChoice')){
                 choiceText = choice.val();
             }else{
-                return;//not available yet
+                return true;//not available yet
```

Comment:
Not sure about this one. Your implementation exits early; the previous one with `_.each()` would exit early only with `return false`. Perhaps it should continue iterating?